templates/skeleton/app/routes/account/login.tsx

```diff
@@ -74,3 +66,25 @@ export default function Login() {
     </Form>
   );
 }
+
+// Remix v1 implementation of ErrorBoundary; for Remix v2, this will need to be updated according to the following documentation:
+// https://remix.run/docs/en/1.15.0/pages/v2#catchboundary-and-errorboundary
+export const ErrorBoundary: ErrorBoundaryComponent = ({error}) => {
+  console.error(error);
+
+  return <div>There was an error.</div>;
+};
+
+// Remix v1 implementation of CatchBoundary; for v2, you can remove `CatchBoundary` and implement the error handling in `ErrorBoundary`:
+// https://remix.run/docs/en/1.15.0/pages/v2#catchboundary-and-errorboundary
+export function CatchBoundary() {
```

Comment:
Kind of the same inconsistency as above. It would be great to get `CatchBoundary` and `ErrorBoundary` aligned signature wise. e.g both `const` or both `function`

Reply:
I won't harp on this anymore since v2 is so close anyway, but thought we'd aligned on embracing stable v2 future flags — which would mean getting rid of catch boundary